build/l10n-pseudo.js

```diff
@@ -0,0 +1,143 @@
+'use strict';
```

Comment:
#3 +1
Well, as far as I'm concerned, I prefer to put all l10n related scripts in build/l10n/ folder and keep multilocale.js in build. Since build/l10n.js, shared/js/l10n.js and even l10n-pseudo.js are unrelated to build system, I'd like to put them in buildl10n/ together or so far as to move out of build/ to separate out of l10n logic from build system.

Quick question, what's difference between build/l10n.js and shared/js/l10n.js?